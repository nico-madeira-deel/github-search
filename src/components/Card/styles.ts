import styled, { css } from 'styled-components'

export const Wrapper = styled.section`
  width: 100%;

  ${({ theme }) => css`
    background-color: ${theme.colors.white};
    color: ${theme.colors.darkBlue};
    border-radius: ${theme.border.radius};
    margin: ${theme.spacings.medium} 0;
    padding: ${theme.spacings.small};
  `}
`
