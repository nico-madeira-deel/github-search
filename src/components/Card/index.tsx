import React from 'react'
import * as S from './styles'

type CardProps = {
  children: React.ReactNode
}

const Card = ({ children }: CardProps) => <S.Wrapper>{children}</S.Wrapper>

export default Card
