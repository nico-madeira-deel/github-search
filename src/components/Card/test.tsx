import Card from '.'
import { renderWithTheme } from 'utils/tests/helpers'

describe('<Card />', () => {
  it('should render the heading', () => {
    const { container } = renderWithTheme(<Card />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
