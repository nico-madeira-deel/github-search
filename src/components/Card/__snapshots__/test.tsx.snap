// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> should render the heading 1`] = `
.c0 {
  width: 100%;
  background-color: #faf9f9;
  color: #555b6e;
  border-radius: 0.5rem;
  margin: 3.2rem 0;
  padding: 2.4rem;
}

<section
  class="c0"
/>
`
